loafwallet/TransactionsViewController.swift

```diff
@@ -57,10 +57,6 @@ class TransactionsViewController: UIViewController, UITableViewDelegate, UITable
     override func viewDidLoad() {
       setup()
       addSubscriptions()
-
```

Comment:
I'm pretty sure I added this because the DispatchQueue  was recommended.
But you removed the            
` self.currentPromptType = .recommendRescan`
 From my recollection means the PromptType is nil.
 
 Is there a  reason why you took it out completely?

Reply:
This is just a part of a code that I wrote and forgot to remove in the first commit so I did it in the second. It just helped me to trigger a rescan recommended prompt to find out what's wrong. Compared to develop branch, this code did not exist.